virtual-desktop/src/app/window-manager/mvd-window-manager/shared/window-manager.service.ts

```diff
@@ -70,6 +71,7 @@ export class WindowManagerService implements MVDWindowManagement.WindowManagerSe
   private viewportManager: MVDHosting.ViewportManagerInterface;
   private pluginManager: MVDHosting.PluginManagerInterface;
   public screenshotRequestEmitter: Subject<{pluginId: string, windowId: MVDWindowManagement.WindowId}>; 
+  private authenticationManager: any;
```

Comment:
It is possible that by altering the type info here that it could break plugins that currently use it.
I don't want to touch it at this time, please. I know this wont cause a runtime issue but it could cause a built time issue.

Reply:
Nevermind, Lenny's right, this can safely be not 'any'.